DB/GADPerson.h

```diff
@@ -51,10 +51,10 @@ typedef enum {Student, FacStaff, SGA} GADPersonType;
  
  Inside the completion handler, there's a GADPerson Array called people (you can name it whatever you want actually), containing the results the url returns for you to use.
  */
-+(void) fetchPersonInfoWithCriteria:(NSDictionary*)criteria
-                           Username:(NSString*)username
-                           Password:(NSString*) password
-                  completionHandler:(void(^_Nonnull)(NSArray<GADPerson *> *))completion;
++(void) executeWithDict:(NSDictionary*)criteria
```

Comment:
Do we want this function to be public anymore? We want the front end to retrieve `GADPerson`s through `-[GADQuery executeWith...]`, right?